static_src/util/cf_api.js

```diff
@@ -423,6 +423,34 @@ export default {
   },
 
   // TODO refactor with org user permissions
+  postCreateNewUserWithGuid(userGuid) {
+    // return http.put(`${APIV}/users/${userGuid}/${role}/${orgGuid}/`)
+    return http.post(`${APIV}/users`, {
+      guid: userGuid
+    })
+      .then(res => this.formatSplitResponse(res.data))
+      .catch(res => {
+        if (res && res.response && res.response.status === 400) {
+          // The user is unauthenicated.
+          return Promise.resolve({ guid: userGuid });
```

Comment:
At this point, how does the caller of this function know if a new user was created or if it was a 400 response?